jenkins/bootstrap_test.py

```diff
@@ -1268,7 +1268,16 @@ def testBootstrapCIYaml(self):
             'kubernetes-e2e-gke-1.3-1.4-upgrade-master',
             'kubernetes-e2e-gce-1.3-1.4-upgrade-master',
             'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster',
-            'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new'
+            'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new',
```

Comment:
Will remove these gce upgrade jobs here after all of them have been merged, and modify the blacklist pattern.